package.json

```diff
@@ -2,18 +2,25 @@
   "name": "@geoblocks/sources",
   "version": "0.1.3",
   "description": "Geoblocks sources",
-  "module": "src/index.js",
+  "main": "dist/sources.js",
+  "module": "dist/sources.js",
+  "source": "src/index.js",
   "scripts": {
     "eslint": "eslint src/*.js",
     "eslint-fix": "eslint src/*.js --fix",
-    "build": "webpack --mode production",
-    "build-debug": "webpack --mode development",
+    "build": "microbundle --format es",
+    "build-example": "webpack --mode production",
+    "build-example-debug": "webpack --mode development",
     "start": "webpack-dev-server --mode development --content-base dist/ --watch",
     "typecheck": "tsc --pretty",
     "lint": "npm run eslint && npm run typecheck",
-    "test": "npm run lint && npm run build && npm run build-debug",
+    "test": "npm run lint && npm run build-example && npm run build-example-debug",
```

Comment:
In order for the microbundle build to be tested by travis, you should also add it here, right?

Reply:
Yes, good point, I will add it.